perf.groovy

```diff
@@ -341,15 +408,18 @@ def static getOSGroup(def os) {
                 }
             }
             def osGroup = getOSGroup(os)
-            def architecture = 'x64'
-            def configuration = 'Release'
             def runType = isPR ? 'private' : 'rolling'
             def benchViewName = isPR ? 'coreclr private \$BenchviewCommitName' : 'coreclr rolling \$GIT_BRANCH_WITHOUT_ORIGIN \$GIT_COMMIT'
 
             steps {
                 shell("bash ./tests/scripts/perf-prep.sh --throughput")
```

Comment:
I don't think building on RHEL should have an impact should it?  Our goal is to be as close to shipping as possible, which happens on RHEL.  Ideally, we don't even build CoreFX separately - we use the test layout logic that gets used in functional runs.

As far as the baseline, we did actually just update it last week because it was old.  What we should do to keep from having the stale baseline problem is take a set of binaries and make them the "curated set of binaries" to be used on throughput and don't depend on using framework binaries.  That way, we don't have to worry about any framework binaries that get treated specially or types that might get forwarded into System.Private.CoreLib since S.P.C will continue to move forward alongside the runtime used to run TP tests.